reducers/comments.js

```diff
@@ -23,6 +23,7 @@ export default function comments( state = defaultState, action ) {
 			}
 		case 'COMMENTS_UPDATING':
 			state.list.loading = true
+			state.list.lastError = null;
```

Comment:
Can we change this to a spread rather than mutation while we're here?

Reply:
Will do this in a follow up for all reducers